client/src/assets/css/styles.css

```diff
@@ -527,26 +529,50 @@ table {
    
  }
  
+ .wavecontainer {
```

Comment:
Add placeholder for container as this is primary wrapper around waves and slogan